cwms-data-api/src/test/java/cwms/cda/data/dao/location/kind/ProjectStructureIT.java

```diff
@@ -146,4 +157,52 @@ public static void deleteLocation(DSLContext context, String officeId, String lo
 			LOGGER.atFinest().withCause(ex).log("No data found for " + officeId + "." + locId);
 		}
 	}
+
+	public static void deleteLocationGroup(DSLContext context, Outlet outlet) {
+		LocationGroupDao locationGroupDao = new LocationGroupDao(context);
+		try {
+			locationGroupDao.delete(outlet.getRatingCategoryId().getName(), outlet.getRatingGroupId().getName(), true, OFFICE_ID);
+		} catch (NotFoundException e) {
+			LOGGER.atFinest().withCause(e).log("No data found for category:" + outlet.getRatingCategoryId().getName()
+													   + ", group-id:" + outlet.getRatingGroupId().getName());
+		}
+	}
+
+	public static <T extends CwmsDTOBase> void containsDto(List<T> outlets, T expectedDto, BiPredicate<T, T> identifier, BiConsumer<T, T> dtoMatcher) {
+		T receivedOutlet = outlets.stream()
+									   .filter(dto -> identifier.test(dto, expectedDto))
+									   .findFirst()
+									   .orElse(null);
+		assertNotNull(receivedOutlet);
+		dtoMatcher.accept(expectedDto, receivedOutlet);
+	}
+
+	public static <T extends CwmsDTOBase> void doesNotContainDto(List<T> outlets, T expectedDto, BiPredicate<T, T> identifier) {
+		T receivedDto = outlets.stream()
+									   .filter(dto -> identifier.test(dto, expectedDto))
+									   .findFirst()
+									   .orElse(null);
+		assertNull(receivedDto);
+	}
+
+	public static void createRatingSpecForOutlet(DSLContext context, Outlet outlet, String specId) {
```

Comment:
Move outlet code to implementation class instead of base class of turbines and outlets. Also rename occurrences of `outlets` in the file to avoid miscommunication.

Reply:
I moved these into a base class, so I can share them across the extended Outlet classes

Reply:
and not worth creating an OutletStructureIT base class, I'll buy it!

Reply:
oh, looks like you did make one